tests/test-cases/Group1-Docker-Commands/1-14-Docker-Kill.robot

```diff
@@ -96,9 +102,9 @@ Confirm signal delivered to entire process group
     Should Be Equal As Integers  ${rc}  1
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} kill -s HUP ${id}
```

Comment:
We may want to wait for the HUP signal to be confirmed before we call stop:
```
Wait Until Keyword Succeeds  20x  200 milliseconds  Assert In Container Output  ${id}  KillSignalHUP
```
THEN we stop it and assert that the child didn't print output.